src/greptimeai/collection.py

```diff
@@ -423,18 +429,16 @@ def _do_start_span(ctx: Optional[Context] = None):
             self._trace_tables.put_trace_context(run_id, trace_context)
 
         if not run_id:
-            logger.error("unexpected behavior. run_id MUST NOT be empty to start_span")
+            logger.error("unexpected behavior of start_span. run_id MUST NOT be empty.")
             return
 
         if parent_run_id:
-            trace_context = self._trace_tables.get_trace_context(
-                parent_run_id, span_name
-            )
+            trace_context = self._trace_tables.get_trace_context(parent_run_id)
```

Comment:
this is what this PR wants to fix. Parent span's name is likely to be different from self.